.github/workflows/falcon_configure.yml

```diff
@@ -48,7 +48,7 @@ jobs:
         run: |
           sudo apt install apt-transport-https ca-certificates curl software-properties-common
           python -m pip install --upgrade pip
-          pip install molecule[docker,lint,ansible] yamllint ansible-lint
+          pip install molecule[docker,lint,ansible] yamllint ansible-lint==5.4.0
```

Comment:
There is currently a bug in ansible-lint 6.0+ with ansible deps and other python deps we use. This current version works, however we should further investigate why 6.0+ is not working correctly for our stuff.